### PR TITLE
Update package versions: RADICAL-Cybertools (RE, RG, RP, RS, RU)

### PR DESCRIPTION
**Pull request by mtitov**
RADICAL-EnTK, RADICAL-GTOD, RADICAL-Pilot, RADICAL-SAGA, RADICAL-Utils

### CODE REVIEW

**Review by tldahlgren — APPROVED** [review]
Confirmed all new sha256.  Deferring merge decision until maintainer and @adamjstewart have a chance to comment.

File: var/spack/repos/builtin/packages/py-radical-gtod/package.py [hunk 1]
```diff
@@ -14,12 +14,15 @@ class PyRadicalGtod(PythonPackage):
...
 
     maintainers = ['andre-merzky']
 
     version('develop', branch='devel')
+    version('1.13.0',  sha256='15df4ae728a8878b111cfdedffb9457aecc8003c2cfbdf2c918dfcb6b836cc93')
     version('1.6.7',   sha256='8d7d32e3d0bcf6d7cf176454a9892a46919b03e1ed96bee389380e6d75d6eff8')
 
-    depends_on('python@3.6:',   type=('build', 'run'))
-    depends_on('py-setuptools', type='build')
+    depends_on('py-radical-utils', type=('build', 'run'), when='@1.13:')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-radical-gtod/package.py:25]: Where do you see this dependency listed?

Comment (mtitov) [reply var/spack/repos/builtin/packages/py-radical-gtod/package.py:25]: That dependency listing was missed, and currently it is in the devel-branch (will be in coming fix or new release), but dependency itself is should be set here

**Review by andre-merzky — APPROVED** [review]
LGTM, thanks!

File: var/spack/repos/builtin/packages/py-radical-gtod/package.py (hunk 1 above)
Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-radical-gtod/package.py:25]: In that case, should it be `when='@1.14:'`? I don't see it imported in 1.13.

Comment (mtitov) [reply var/spack/repos/builtin/packages/py-radical-gtod/package.py:25]: it is used to extract package version - https://github.com/radical-cybertools/radical.gtod/blob/v1.13.0/src/radical/gtod/__init__.py

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-radical-gtod/package.py:25]: Thanks, must have missed that somehow.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**